WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -787,6 +797,14 @@ private void disableAndRemovePlugin() {
     public void onSitePluginConfigured(OnSitePluginConfigured event) {
         if (isFinishing()) return;
 
+        if (mSite.getId() != event.site.getId() // Wrong site
+                || !mPlugin.isInstalled() // Plugin is not installed, this event can't be about this plugin
+                || mPlugin.getName() == null // Sanity check for NPE, but if the plugin is installed name will be there
+                || !mPlugin.getName().equals(event.pluginName)) { // Checking if the configured plugin is the one we are showing
+            // Not the event we are interested in
+            return;
+        }
```

Comment:
This same conditional is used a few times in the class.  We could make it into a method named something like `checkSitePlugin` to avoid bugs caused by changing just one instance.  What do you think?

Reply:
As discussed on Slack, I think this would be easier to do once we have this PR merged in so that we can utilize the `slug` property for all the checks. https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/722

I've added this item to #7263 and will take care of it before these changes are merged to `develop`.

Thanks again for the review!